drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -526,15 +529,37 @@ local function initialize_switch(driver, device)
     device:set_field(COMPONENT_TO_ENDPOINT_MAP_BUTTON, component_map, {persist = true})
   end
 
-  if num_switch_server_eps > 0 then
+  if #switch_eps > 0 and #button_eps > 0 then
+    if tbl_contains(STATIC_BUTTON_PROFILE_SUPPORTED, #button_eps) then
+      local dimmable_light = false
+      for _, ep in ipairs(device.endpoints) do
+        for _, dt in ipairs(ep.device_types) do
+          if dt.device_type_id == DIMMABLE_LIGHT_DEVICE_TYPE_ID then
+            dimmable_light = true
```

Comment:
```suggestion
            profile_name = "light-level"
```

Reply:
Updated as suggested in 96e319121530b6b2ad5ff7114b02531588b3bdcd